src/app/beer_garden/db/mongo/api.py

```diff
@@ -279,7 +289,7 @@ def create(obj: ModelItem) -> ModelItem:
     if hasattr(mongo_obj, "deep_save"):
         mongo_obj.deep_save()
     else:
-        mongo_obj.save()
+        mongo_obj.save(force_insert=True)
```

Comment:
They're unrelated. `deep_save` is something that we made up so that saving a `System` will save the nested models (instances and commands) as well.

This `force_insert` flag makes a save operation raise an exception if that model already exists in the database. Without it "save" basically means "save or update".

Reply:
sounds good